arch/Kconfig

```diff
@@ -347,56 +347,56 @@ config ARCH_HAS_THREAD_ABORT
 # Hidden PM feature configs which are to be selected by
 # individual SoC.
 #
-config SYS_POWER_LOW_POWER_STATES_SUPPORTED
+config SYS_POWER_HAS_LOW_POWER_STATES
 	# Hidden
 	bool
 	help
 	  This option signifies that the target supports the SYS_POWER_LOW_POWER_STATES
 	  configuration option.
 
-config SYS_POWER_STATE_CPU_LPS_1_SUPPORTED
+config SYS_POWER_STATE_LOW_POWER_1_SUPPORTED
```

Comment:
shouldn't this be SYS_POWER_HAS_LOW_POWER_1 or something lie that?

Reply:
Sorry, fixed.